### PR TITLE
Update getting-therapy.md

### PR DESCRIPTION
**Pull request by kimbod**
Clinician feedback

### CODE REVIEW

File: depression-and-anxiety/getting-therapy.md [hunk 1]
```diff
@@ -13,14 +13,16 @@ Talking to a therapist can help with:
 - finding ways of coping every day
 
 Usually treatment for depression and anxiety can include: talking to a therapist, medication, and self-help such as exercise.
-
+a
```
Comment (evilstreak) [inline depression-and-anxiety/getting-therapy.md:16]: There's a stray character here

File: depression-and-anxiety/getting-therapy.md [hunk 2]
```diff
@@ -13,14 +13,16 @@ Talking to a therapist can help with:
...
 There are many different types of talking treatments.  The charity Mind has more information about [types of therapy](http://www.mind.org.uk/information-support/drugs-and-treatments/talking-treatments/types-of-therapy/#.VmBFKHbhCM8). 
 
 ##How to get therapy on the NHS
 
 1. Speak to your GP about the way you feel.
-2. If your GP thinks therapy is the best treatment for you they’ll give you the number for a service that provides therapy in your area.
-3. You phone the local service, they’ll ask you some questions and refer you to one of their therapists.
+2. If your GP thinks therapy is the best treatment, they’ll either refer for therapy in your area or give you the number for a service that provides therapy in your area.
```
Comment (evilstreak) [inline depression-and-anxiety/getting-therapy.md:22]: Should this read "refer **you** for therapy"?

Comment (pacharanero) [inline depression-and-anxiety/getting-therapy.md:22]: On 7 December 2015 at 12:03, Dominic Baggott notifications@github.com
wrote:

> Should this read "refer _you_ for therapy"?

​yes it should I reckon​

File: depression-and-anxiety/getting-therapy.md (hunk 1 above)
Comment (fawkesley) [inline depression-and-anxiety/getting-therapy.md:16]: fixed

File: depression-and-anxiety/getting-therapy.md (hunk 2 above)
Comment (fawkesley) [inline depression-and-anxiety/getting-therapy.md:22]: fixed

**Pull request closed without merging**